### PR TITLE
LHD skewing fix, take two

### PR DESCRIPTION
**Pull request by BadiBasso**
PR should be ok now.
Ref #38

### CODE REVIEW

**Comment by BadiBasso** [comment]
Off topic: my Steam nic is LeoMeisseli.

File: ParallelRoadTool/Detours/NetManagerDetour.cs [hunk 1]
```diff
@@ -348,7 +355,14 @@ private static ushort NodeAtPositionOrNew(ref Randomizer randomizer, NetInfo inf
...
                         Singleton<SimulationManager>.instance.m_currentBuildIndex + 1,
                         Singleton<SimulationManager>.instance.m_currentBuildIndex, invert);
-                }                
+                }
+                // Left-hand drive revert conditions back
+                if (ParallelRoadTool.Instance.IsLeftHandTraffic)
+                {
+                    invert = !invert;
+                    isReversed = !isReversed;
+                    _isPreviousInvert = invert;
```
Comment (ST-Apps) [inline ParallelRoadTool/Detours/NetManagerDetour.cs]: Shouldn't we revert `ParallelRoadTool.Instance.IsSnappingEnabled = true;` too and setting it to false?

**Comment by ST-Apps** [comment]
Ok, it kinda works but we have two issues:

1) Overlay starting nodes (the ones in red) are shown on the wrong side.


2) Using `ParallelRoadTool.Instance.IsSnappingEnabled = true;` enables snapping to other nodes, so the elevated segments in the picture end up snapping to the ground level ones.

**Comment by ST-Apps** [comment]
@BadiBasso ok, something messed up during merge but it should be fixed now. Can you please download branch [ISSUE-38-2](https://github.com/ST-Apps/CS-ParallelRoadTool/tree/ISSUE-38-2) and see if it behaves as you'd expect? Thanks.

**Comment by BadiBasso** [comment]
Seems ok. No snapping when crossing over or under existing roads Overlay is on the right side.
lhd capture

**Comment by ST-Apps** [comment]
Fine, released it in the beta branch to get some feedback from users.
Thanks for your help :)

**Comment by BadiBasso** [comment]
Concerning user FuzzyOrb's comment in Steam's beta, the reason is this line:
`//disable collision on current network  
selectedNetInfo.m_canCollide = false;`
LHD works just as fine without it, so what if this setting would be selectable by a checkbox. What do you think?

**Comment by ST-Apps** [comment]
I'd remove it as this is managed by both Road Anarchy and Prop & Tree Anarchy, so there's no point in having a duplicate functionality.

**Comment by BadiBasso** [comment]
Fine with me. Less confusion then, I suppose

**Pull request merged**